AppCenter/AppCenter/Internals/Vendor/Reachability/MS_Reachability.m

```diff
@@ -148,13 +148,13 @@ - (void)stopNotifier {
 }
 
 - (void)dealloc {
-  __block SCNetworkReachabilityRef reachabilityRef = self.reachabilityRef;
-  if (reachabilityRef != NULL) {
-    [MSDispatcherUtil performBlockOnMainThread:^{
-      SCNetworkReachabilityUnscheduleFromRunLoop(reachabilityRef, CFRunLoopGetCurrent(), kCFRunLoopDefaultMode);
-      CFRelease(reachabilityRef);
-    }];
-  }
+    __block SCNetworkReachabilityRef reachabilityRef = self.reachabilityRef;
+    if (reachabilityRef != NULL) {
+      [MSDispatcherUtil performBlockOnMainThread:^{
+        SCNetworkReachabilityUnscheduleFromRunLoop(reachabilityRef, CFRunLoopGetCurrent(), kCFRunLoopDefaultMode);
+        CFRelease(reachabilityRef);
+      }];
+    }
```

Comment:
This formatting change doesn't seem correct

Reply:
Fixed